services/src/bin/main.rs

```diff
@@ -45,25 +53,40 @@ fn initialize_logging() -> Result<LoggerHandle> {
             file_spec = file_spec.directory(dir);
         }
 
-        logger = logger
-            .log_to_file(file_spec)
-            .write_mode(if logging_config.enable_buffering {
-                WriteMode::BufferAndFlush
-            } else {
-                WriteMode::Direct
-            })
+        // TODO: use local time instead of UTC
+        // On Unix, using local time implies using an unsound feature: https://docs.rs/flexi_logger/latest/flexi_logger/error_info/index.html#time
+        // Thus, we use UTC time instead.
+        flexi_logger::DeferredNow::force_utc();
+
+        let (file_writer, fw_handle) = FileLogWriter::builder(file_spec)
+            .write_mode(WriteMode::Async)
             .append()
             .rotate(
                 Criterion::Age(Age::Day),
                 Naming::Timestamps,
                 Cleanup::KeepLogFiles(7),
             )
-            .duplicate_to_stderr(Duplicate::All);
+            .try_build_with_handle()
+            .unwrap();
+
+        let file_layer = tracing_subscriber::fmt::layer()
+            .with_file(false)
```

Comment:
can we avoid specifying the output format (target, file) again? cf. L41ff

Reply:
The layer isn't `Clone` or `Copy`, so I don't know how.